### PR TITLE
Only show body content

### PR DESCRIPTION
**Pull request by yihyunjoon**
Screen Shot 2021-04-25 at 2 54 32 AM

### CODE REVIEW

**Comment by yihyunjoon** [comment]
#18

**Comment by corona10** [comment]
- Once https://github.com/rust-kr/browser-engineering/pull/26 is merged, would you like to write the unitest for `lex`?
- Please add your name at https://github.com/rust-kr/browser-engineering/blob/master/README.md
- Please add your name at https://github.com/rust-kr/browser-engineering/blob/master/rust/Cargo.toml
- Please check https://github.com/rust-kr/browser-engineering/pull/27/checks?check_run_id=2429787063

File: rust/src/lib.rs [hunk 1]
```diff
@@ -217,19 +217,31 @@ Accept-Encoding: gzip,deflate\r
     pub fn lex(body: &[u8]) -> String {
         // 13. Print content
         let mut in_angle = false;
-        let mut ret = String::new();
+        let mut in_body_tag = false;
+        let mut out = String::new();
+        let mut tag = String::new();
```
Comment (youknowone) [inline rust/src/lib.rs:257]: because `body` is alive until the function end, `out` and `tag` don't need to be `String` but to be `&[u8]`.

**Review by youknowone — CHANGES_REQUESTED** [review]

**Comment by yihyunjoon** [comment]
> * Once #26 is merged, would you like to write the unitest for `lex`?

Sure, I'll do it later.

> * Please add your name at [https://github.com/rust-kr/browser-engineering/blob/master/README.md](https://github.com/rust-kr/browser-engineering/blob/master/README.md?rgh-link-date=2021-04-25T03%3A26%3A03Z)

I didn't add my name in `README.md`. I haven't decided what chapter to do yet.

> * Please add your name at [https://github.com/rust-kr/browser-engineering/blob/master/rust/Cargo.toml](https://github.com/rust-kr/browser-engineering/blob/master/rust/Cargo.toml?rgh-link-date=2021-04-25T03%3A26%3A03Z)

Added!

> * Please check [#27 (checks)](https://github.com/rust-kr/browser-engineering/pull/27/checks?check_run_id=2429787063)

Formatted the code. It will pass.

File: rust/src/lib.rs (hunk 1 above)
Comment (yihyunjoon) [reply rust/src/lib.rs:257]: I'll do this on another branch.

**Comment by corona10** [comment]
> I didn't add my name in README.md. I haven't decided what chapter to do yet.

This task is part of chapter 1

File: rust/src/lib.rs (hunk 1 above)
Comment (corona10) [reply rust/src/lib.rs:257]: Please follow code review if possible in a single PR.

**Pull request merged**